clang/www/features.html

```diff
@@ -28,7 +28,7 @@ <h1>Clang - Features and Goals</h1>
 <p>End-User Features:</p>
 
 <ul>
-<li><a href="#performance">Fast compiles and low memory use</a></li>
+<li><a href="#performance">Fast Compile-times and low memory use</a></li>
```

Comment:
"fast compile-times" is probably correct English but reads awkwardly for me, and is a bit on the informal side for technical documentation.

Instead you could change "compiles" to "compilation". This is a simpler overall.